docs/content/en/docs/Advanced/partition_encryption.md

```diff
@@ -0,0 +1,286 @@
+---
+title: "Encrypting User Data with Kairos"
+linkTitle: "Encrypting User Data with Kairos"
+weight: 5
+description: >
+    This section describes how to encrypt partition with LUKS in Kairos.
+---
+
+{{% alert title="Note" color="warning" %}}
+
+This feature will be available in Kairos version `1.5.0` and in all future releases.
+
+{{% /alert %}}
+
+Kairos offers the ability to encrypt user data partitions with `LUKS`. User-data partitions are dedicated to persist data for a running system, stored separately from the OS images. This encryption mechanism can also be used to encrypt additional partitions created during the installation process.  
+
+Kairos supports the following encryption scenarios:  
+
+1. **Offline mode** - Encryption key for partitions is stored on the machine inside the TPM chip.  
+1. **Online mode (Automated)** - Keypair used to encrypt the partition passphrase is stored on the TPM chip.
```

Comment:
In this short view, I cant discern between the ofline and online(automated) modes easily. Maybe a more precise description or and advantage or something? 

Its clear to me comparing to the online(manual) config at a glance what it entails, not so much with the other 2.

Reply:
good point! changed in https://github.com/kairos-io/kairos/pull/722/commits/f56a73669d480298f9f73c6f6db5608855264128